src/main/java/seedu/exercise/ui/ResourceListPanel.java

```diff
@@ -60,11 +60,14 @@ private void setResourceListListener() {
             public void onChanged(Change<? extends Resource> change) {
                 while (change.next()) {
                     int index = -1;
+                    /*
                     if (change.wasReplaced()) {
                         index = change.getFrom();
                     } else if (change.wasAdded()) {
                         index = resourceList.size() - 1;
                     }
+                     */
```

Comment:
If this chunk of code is not required, please just take it out instead of commenting it out

Reply:
Okay